Marlin/Marlin_main.cpp

```diff
@@ -7413,6 +7554,20 @@ void process_next_command() {
           gcode_M0_M1();
           break;
       #endif // ULTIPANEL
+     
+      #ifdef SPINDLE_ENABLED
+        case 3:
+          gcode_M3();
```

Comment:
Following current conventions, this function should be called `gcode_M3_M4` instead.